packages/web-app-files/src/components/Search/List.vue

```diff
@@ -290,13 +290,26 @@ export default defineComponent({
     watch(
       () => unref(route).query,
       (newVal, oldVal) => {
-        const filters = ['q_fullText', 'q_tags', 'useScope']
-        const isChange =
-          newVal?.term !== oldVal?.term ||
-          filters.some((f) => newVal[f] ?? undefined !== oldVal[f] ?? undefined)
-        if (isChange && isLocationCommonActive(router, 'files-common-search')) {
-          emit('search', buildSearchTerm(true))
+        // return early if this view is not active, no search needed
+        {
+          const isSearchViewPainted = isLocationCommonActive(router, 'files-common-search')
```

Comment:
Painted ?

Reply:
yup, painted, e.g. painted on screen, rendered, visible, ...

Reply:
you like active better?

Reply:
"Mounted", as per Vue-jargon?

Reply:
i updated it to `isSearchViewActive` since the router guard is also called `isLocationCommonActive`, thanks guys for the input